src/omv/framebuffer.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2013/2014 Ibrahim Abdelkader <i.abdalkader@gmail.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * Framebuffer stuff.
+ *
+ */
+#include "imlib.h"
+#include "omv_boardconfig.h"
+#include "framebuffer.h"
+// If buffer size is bigger than this threshold, the quality is reduced.
+// This is only used for JPEG images sent to the IDE not normal compression.
+#define JPEG_QUALITY_THRESH     (160*120*2)
+
+extern char _fb_base;
+framebuffer_t *fb = (framebuffer_t *) &_fb_base;
```

Comment:
I don't understand why you are requesting this. The previous method was creating a static pointer var in each c file the header was included in that all pointed to the same frame buffer and jpeg buffer. My moving these values into their own C file I'm freeing up a lot of .data waste. As for multiple frame buffers... not sure I follow you there. Is this something we need to support now? I'm just trying to do a re-factor here. Not change how everything works. Anyway, changing the name of the stuff from "fb" will break other code in the firmware. There are quite a few places that don't use MAIN_FB().